dockers/Screenshotter/screenshotter.js

```diff
@@ -494,10 +505,14 @@ function takeScreenshot(key) {
                               // corners
                     diffFile, // output file name
                 ]);
-            })
-            .then(function() {
+            });
+        }
+        if (!opts.regenerate) {
+            promise = promise.then(function() {
                 return promisify(fs.unlink, bufFile);
```

Comment:
I thought this would end up deleting the `new` file when `--regenerate` isn't passed, which would have been a bit unexpected, but it just deletes it within the docker.  Nice!